lib/modules/manager/puppet/extract.spec.ts

```diff
@@ -0,0 +1,159 @@
+import { Fixtures } from '../../../../test/fixtures';
+import { GithubTagsDatasource } from '../../datasource/github-tags';
+import { PuppetForgeDatasource } from '../../datasource/puppet-forge';
+import { extractPackageFile } from './extract';
```

Comment:
```suggestion
import { extractPackageFile } from '.';
```

code coverage 😉

Reply:
I do not understand what this has to do with code coverage (less characters to cover?), but done ;)

Reply:
otherwise the `extractPackageFile` export from index file is not called / covered and ci will fail 😉